example/js/scalableTable.js

```diff
@@ -0,0 +1,840 @@
+angular.module('scalable-table', [])
```

Comment:
@myfoxtail I thought that it will be easier to the user to run the example. It don't requires from an user path to URl which looks like http://some-url/example. But other side it can confuse user. So maybe it needs to changes. Is it OK if I'll remove this file from example directory and use main directive?